src/lib/bcmp/bm/bm_printf.h

```diff
@@ -19,7 +19,7 @@ typedef enum {
 } bm_printf_err_t;
 
 bm_printf_err_t bm_fprintf(uint64_t target_node_id, const char* file_name,
-                           bool print_time, const char* format, ...);
+                           uint8_t print_time, const char* format, ...);
```

Comment:
Also changed this to a `uint8_t` to be in alignment with the MACROs associated above/struct assignment in function definition